src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
+$cell-size: 100px;
```

Comment:
If we do this task according to Sass and BEM, we need to create separate files and folders for _variables, blocks...